File: Web/sneha-jais_snehaJaiswal_2024cs1084_2/gettingstarted/digits_of_number.cpp
#include <iostream>
using namespace std;

void sum(int x){
   if(x>=10)
      sum(x/10);
      
      int digit=x%10;
      cout<<digit<<" ";
   
    }

int main(int argc, char **argv){
    int n;
    cin >> n;

    //write your code here
   sum(n);
    return 0;
    
}

File: Android/SahilBhragudev_SahilBhragudev_2024cse1116_2/GettingStarted/Digits_of_number.java
import java.util.*;
    
    public class Digits_of_number{
    
    public static void main(String[] args) {
      // write your code here  
      Scanner sc = new Scanner(System.in);
      int n1 = sc.nextInt();
      int digit;
      int n=0,temp;
      while(n1!=0){
          
           temp=n1%10;
           n=n*10+temp;
           n1=n1/10;
         
      }
      while(n!=0)
      {
          digit=n%10;
          System.out.println(digit);
          n=n/10;
      }
     }
    }
